lib/terraforming/resource/route53_record.rb

```diff
@@ -66,18 +73,32 @@ def record_sets_of(hosted_zone)
       end
 
       def records
-        hosted_zones.map do |hosted_zone|
+        to_return = hosted_zones.map do |hosted_zone|
           record_sets_of(hosted_zone).map { |record| { record: record, zone_id: zone_id_of(hosted_zone) } }
         end.flatten
+        count = {}
+        dups = to_return.group_by{ |record| module_name_of(record[:record], nil) }.select { |k, v| v.size > 1 }.map(&:first)
+        to_return.each do |r|
+          module_name = module_name_of(r[:record], nil)
+          if dups.include?(module_name)
```

Comment:
__[rubocop]__
* Use `next` to skip iteration. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)

<!-- Comment by [SideCI](https://sideci.com) -->